test/unit/fs-emission-symlink/input.js

```diff
@@ -0,0 +1,3 @@
+const fs = require('fs');
+const { join } = require('path');
+fs.readFile(join(__dirname, 'symlink'));
```

Comment:
I think we're looking to capture the `require` test here rather than an asset emission:

```js
require('./symlink')
```

where `symlink` is a symlink to another CommonJS file

Reply:
You’re absolutely right!

I will fix the test tomorrow unless you want to do it before then 🙏

Reply:
I tried to edit on my phone, we’ll see if that worked 🤞